test/API/CMake/CheckUbsan.cmake

```diff
@@ -0,0 +1,37 @@
+set(UBSAN_FLAG "-fsanitize=undefined")
```

Comment:
This too, it would be better to augment saniitzers.cmake for an special settings based on the HDF5_TEST_API value.